components/ui/GraphList/GraphList.tsx

```diff
@@ -0,0 +1,80 @@
+'use-client'
+
+import Graph from '@/components/ui/Graph/Graph';
+import React from 'react';
+
+const GraphList: React.FC = () => {
+  const graphConfig = {
```

Comment:
Here also we can move the `graphConfig` and `graphData` constants to the constant file I stated earlier, i.e. `graph-constants.ts` file. Then import these here, that will make this component better.